x-pack/plugins/secops/public/components/timeline/body/renderers/suricata_signature.tsx

```diff
@@ -80,16 +88,19 @@ export const SuricataSignature = pure<{ id: string; signature: string; signature
   ({ id, signature, signatureId }) => {
     const tokens = getBeginningTokens(signature);
     return (
-      <EuiFlexGroup justifyContent="center" gutterSize="none">
+      <EuiFlexGroup justifyContent="center" gutterSize="none" wrap={true}>
         <DraggableSignatureId id={id} signatureId={signatureId} />
         <Tokens tokens={tokens} />
         <LinkFlexItem grow={false}>
-          <GoogleLink link={signature}>
-            {signature
-              .split(' ')
-              .splice(tokens.length)
-              .join(' ')}
-          </GoogleLink>
+          <div>
+            <GoogleLink link={signature}>
+              {signature
+                .split(' ')
+                .splice(tokens.length)
+                .join(' ')}
+            </GoogleLink>
+            <ExternalLinkIcon />
+          </div>
```

Comment:
Weird. This needed an extra `<div>`? Let me know what the bug is or if this is related to other changes.